code/go/pkg/validator/errors.go

```diff
@@ -1,8 +1,8 @@
 package validator
 
 import (
-	"github.com/elastic/package-spec/code/go/internal/validator"
+	"github.com/elastic/package-spec/code/go/internal/errors"
 )
 
 // ValidationErrors is an Error that contains a iterable collection of validation error messages.
-type ValidationErrors validator.ValidationErrors
+type ValidationErrors errors.ValidationErrors
```

Comment:
I see that it will change the API, but as long as the source code is versioned we should be safe.

Reply:
Yeah, I had to do this to avoid an import cycle. Otherwise I would've preferred to keep the API the same.

If we really want to stay backwards compatible here, we could introduce type aliases like so:

```
package validator

import "github.com/elastic/package-spec/code/go/internal/errors"
  
type ValidationErrors = errors.ValidationErrors
```

However, I think this is overkill considering that, as far as we know, there is nothing that is using this type outside of the `package-spec` module. I checked `elastic-package` and even it does not use it; it simply relies on the `error` interface returned by `validator.ValidateFromPath` instead.

Reply:
Definitely an overkill, let's stick to the current form. We know all consumers of this package.